apps/condo/domains/banking/access/BankIntegrationAccountContext.js

```diff
@@ -7,14 +7,24 @@ const get = require('lodash/get')
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 const { getById } = require('@open-condo/keystone/schema')
 
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/utils/accessSchema')
 const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
 const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
 
-async function canReadBankIntegrationAccountContexts ({ authentication: { item: user } }) {
+const { BANK_INTEGRATION_IDS } = require('../constants')
+
+async function canReadBankIntegrationAccountContexts ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
 
-    if (user.isAdmin) return {}
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === SERVICE) {
+        return {
+            integration: { accessRights_some: { user: { id: user.id }, deletedAt: null } }, deletedAt: null,
```

Comment:
```suggestion
            integration: { accessRights_some: { user: { id: user.id, deletedAt: null }, deletedAt: null } }, deletedAt: null,
```

Reply:
does not make sense, because if the user is deleted, the guard will work at the beginning and the request will be rejected without reaching this check